src/chrome/content/rules/Reuters.com.xml

```diff
@@ -59,6 +59,8 @@
 -->
 <ruleset name="Reuters.com (partial)">
 
+	<target host="reuters.com" />
+	<target host="www.reuters.com" />
```

Comment:
Yes, they WFM. Including `af` and `ca` which are currently listed as non-woring.